LICENSE

```diff
@@ -0,0 +1,373 @@
+Mozilla Public License Version 2.0
```

Comment:
This has been added so let's remove it from your PR

Reply:
Done.